aiida/common/setup.py

```diff
@@ -1070,6 +1079,8 @@ def set_property(name, value):
         actual_value = unicode(value)
     elif type_string == "int":
         actual_value = int(value)
+    elif type_string == 'list_of_str':
+        actual_value = value.replace(',', ' ').split()
```

Comment:
So both spaces and commas are valid separators? If yes, I would add to the documentation above of verdishell.calculation_list. But I would rather reduce the number of valid separators to avoid problems if we want to keep the separator. So either only `split()` or only `split(',')`